redhat-operators/tektoncd-pipeline-operator/tektoncd-pipeline-operator.package.yaml

```diff
@@ -0,0 +1,5 @@
+packageName: tetektoncd-pipeline-operatorktoncd
```

Comment:
Please make sure your packageName matches the operator's directory, ie change the packageName to match the directory name, or vice versa.

Reply:
this is an embarrassing mistake. Thank you for catching it :+1:

Reply:
done